astropy/units/format/vounit.py

```diff
@@ -71,7 +72,9 @@ def do_defines(bases, prefixes, skips=[]):
                         continue
                     if keyword.iskeyword(key):
                         continue
-                    names[key] = getattr(u, key)
+
+                    uwithekey = u if hasattr(u, key) else u_deprecated
```

Comment:
Aha!  In fixing the test failure I managed to shrink this down so it can be in the following line and stay <80.  So no need to worry about the spelling at all! 😛